crates/fuel-core/src/query/contract.rs

```diff
@@ -21,26 +23,35 @@ use fuel_core_types::{
     services::graphql_api::ContractBalance,
 };
 
-pub struct ContractQueryContext<'a>(pub &'a Database);
+pub trait ContractQueryData: Send + Sync {
```

Comment:
Could you add `\n` after each method please?=)

Reply:
Added :)